UI/ConfigWindow.cs

```diff
@@ -1,153 +1,152 @@
 using System;
 using System.Linq;
+using System.Numerics;
 using Dalamud.Game.Text;
 using Dalamud.Interface;
+using Dalamud.Interface.Windowing;
 using ImGuiNET;
 
 namespace DeathRecap.UI;
 
-public class ConfigWindow {
+public class ConfigWindow : Window {
     private readonly DeathRecapPlugin plugin;
 
-    public ConfigWindow(DeathRecapPlugin plugin) {
+    public ConfigWindow(DeathRecapPlugin plugin) : base("Death Recap Config") {
         this.plugin = plugin;
-    }
 
-    public bool ShowConfig { get; internal set; }
+        SizeConstraints = new WindowSizeConstraints
+        {
```

Comment:
1TBS style, but I know how finnicky VS is with formatting, so I can do that later.
```suggestion
        SizeConstraints = new WindowSizeConstraints {
```